.github/workflows/ubi8.yml

```diff
@@ -0,0 +1,64 @@
+name: github-UBI8
+
+on:
+  pull_request:
+    paths-ignore:
```

Comment:
If you want to add a comment about why this is here, it could be helpful for future maintainers. For example:

```
Run the unit tests anytime a code file is changed (does not include documentation files).
```